packages/http-server-javascript/generated-defs/helpers/multipart.ts

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+import { Module } from "../../src/ctx.js";
+
+export let module: Module = undefined as any;
+
+// prettier-ignore
+const lines = [
+  "// Copyright (c) Microsoft Corporation",
+  "// Licensed under the MIT license.",
+  "",
+  "import type * as http from \"node:http\";",
+  "",
+  "export interface HttpPart {",
+  "  headers: { [k: string]: string | undefined };",
```

Comment:
I wonder if the name of the part should be left in the content-disposition header, or captured as a separate property for matching with the expected parts

Reply:
Since this is for multipart generally, not specifically multipart/form-data, I'm not guaranteed that the content-disposition header is present at this stage in the pipeline (e.g. multipart/mixed). Later stages do extract and parse the content-disposition header when it's required.